src/Storages/Statistics/CountMinSketchStatistics.cpp

```diff
@@ -0,0 +1,100 @@
+#include <Storages/Statistics/CountMinSketchStatistics.h>
+#include <DataTypes/DataTypeLowCardinality.h>
+#include <DataTypes/DataTypeNullable.h>
+#include <IO/ReadHelpers.h>
+#include <IO/WriteHelpers.h>
+
+#if USE_DATASKETCHES
+
+namespace DB
+{
+
+namespace ErrorCodes
+{
+extern const int ILLEGAL_STATISTICS;
+}
+
+
+CountMinSketchStatistics::CountMinSketchStatistics(const SingleStatisticsDescription & stat_, DataTypePtr data_type_)
+    : IStatistics(stat_)
+    , sketch(HASH_COUNT, BUCKET_COUNT)
```

Comment:
`HASH_COUNT` --> `num_hashes`, `BUCKET_COUNT` --> `num_buckets` (to use the terminology in the datasketch header in lowercase)